cmd/kubeadm/app/util/pkiutil/pki_helpers.go

```diff
@@ -556,32 +625,39 @@ func NewPrivateKey(keyType x509.PublicKeyAlgorithm) (crypto.Signer, error) {
 }
 
 // NewSignedCert creates a signed certificate using the given CA certificate and key
-func NewSignedCert(cfg *CertConfig, key crypto.Signer, caCert *x509.Certificate, caKey crypto.Signer) (*x509.Certificate, error) {
+func NewSignedCert(isCA bool, cfg *CertConfig, key crypto.Signer, caCert *x509.Certificate, caKey crypto.Signer) (*x509.Certificate, error) {
```

Comment:
this bool should be the last arg ideally.

Reply:
Change made in this commit: https://github.com/kubernetes/kubernetes/pull/97266/commits/6f90532f2db098c52a531bb17e6db39d23a8db41